themes/src/main/resources-community/theme/base/login/messages/messages_fr.properties

```diff
@@ -113,13 +113,13 @@ restartLoginTooltip=Redémarrer la connexion
 
 loginTotpIntro=Il est nécessaire de configurer un générateur One Time Password pour accéder à ce compte
 loginTotpStep1=Installez une des applications suivantes sur votre mobile:
-loginTotpStep2=Ouvrez l''application et scannez le code-barres ou entrez la clef.
+loginTotpStep2=Ouvrez l''application et scannez ce QR code ou entrez la clef.
 loginTotpStep3=Entrez le code à usage unique fourni par l''application et cliquez sur Sauvegarder pour terminer.
 loginTotpStep3DeviceName=Renseignez un nom d''appareil pour vous aider à gérer vos a appareils OTP.
 loginTotpManualStep2=Ouvrez l''application et saisissez la clé
 loginTotpManualStep3=Utilisez la configuration de valeur suivante si l''application permet son édition
 loginTotpUnableToScan=Impossible de scanner?
-loginTotpScanBarcode=Scanner le code barre ?
+loginTotpScanBarcode=Scanner le QR code ?
```

Comment:
```suggestion
loginTotpScanBarcode=Scanner le QR code?
```

Reply:
I saw it too but I kept it how it formerly was for consistency with the other questions (see line 137)

Reply:
Feel free to correct those as well. I can't imagine it's a French tradition to have spaces before question marks, is it? :smile:

Reply:
It actually is 😅 Likewise, we always keep a space BEFORE and AFTER ; : !

Reply:
Well, TIL! Ignore my comments then hahaha